metrics/ci_integrations/test/util/validator/string_validator_test.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:ci_integration/util/validator/string_validator.dart';
+import 'package:test/test.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group("StringValidator", () {
+    Function _validatorCallback(String value, {String name}) {
+      return () => StringValidator.checkNotNullOrEmpty(value, name: name);
+    }
```

Comment:
I'm not sure this callback makes the test cases below more readable, I suppose we should think about inlining this. The reason is that usually wrapping a method call in an empty callback is not required for `Future`s and test cases below can be confusing.